src/Connection.php

```diff
@@ -141,6 +142,18 @@ public function count()
     }
 
     /**
+     * Check if the connection is still active.
+     *
+     * @throws InvalidResourceException If connection was closed
+     *
+     * @return bool
+     */
+    public function isOpen()
```

Comment:
As per [imap_ping](https://secure.php.net/manual/en/function.imap-ping.php) page we can safely add `bool` return type.
Please add this signature to `ConnectionInterface` too.